app/Database/__init__.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import MetaData
+
+convention = {
+    "ix": 'ix_%(column_0_label)s',
+    "uq": "uq_%(table_name)s_%(column_0_name)s",
+    "ck": "ck_%(table_name)s_%(column_0_name)s",
+    "fk": "fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s",
+    "pk": "pk_%(table_name)s"
+}
+
+metadata = MetaData(naming_convention=convention)
+db = SQLAlchemy(metadata=metadata)
+
+class BaseModelMixin:
+    created = db.Column(db.DateTime, nullable=False, default= datetime.utcnow)
+    updated = db.Column(db.DateTime, onupdate = datetime.utcnow)
```

Comment:
```suggestion
    updated = db.Column(db.DateTime, onupdate=datetime.utcnow)
```

Se consistente al momento de codear. Si el resto de tu código lo tienes de una manera, siempre hay que mantenerlo asī en todo el código.